core/esmf-aspect-model-document-generators/src/main/resources/docu/templates/html/common-documentation-lib.vm

```diff
@@ -37,5 +37,5 @@
 #end
 
 #macro ( description $description )
-<span class="text-base py-4">$description</span>
+<span class="text-base py-4">#evaluate( $description )</span>
```

Comment:
Maybe I'm missing something, but where is `evaluate` defined? Should this be `$descriptionUtils.toHtml( $description )`?